### ISSUE TITLE
Fix GitHub pages not serving static images in Quickstart correctly

### ISSUE THREAD

**Issue by ResidentMario** [ResidentMario/geoplot#175]
Unclear why this is happening. [Here is the StackOverflow thread](https://stackoverflow.com/questions/57642081/github-pages-is-not-serving-png-files).

**Comment by jGaboardi** [comment]
[Seem to be rendering correctly now](https://stackoverflow.com/questions/57642081/github-pages-is-not-serving-png-files#comment101735954_57642081).

**Comment by ResidentMario** [comment]
Thanks for the notification @jGaboardi!

This confirms a suspicious I was having, and makes for a really interesting issue.

MacOS preserves case, but is case-insensitive; e.g. it would consider the files `Quickstart.png` and `quickstart.png` to be equivalent filenames, and will use whatever character casing the file had at creation time.

Because I renamed the source file from `Quickstart.ipynb` to `QuickStart.ipynb`, the utility I used to generate these files, `jupyter nbconvert`, moved from outputting the files in title case to outputting them in camel case. so these files were previously title-case (`Quickstart`), and then instead became camel case (`QuickStart`).

`git` attempts to ameliorate this issue, but it's kind of a hack and didn't work in this case, and so this put `git` in a weird state where I couldn't successfully `git add` these files. I had to `git mv` the files onto themselves to move them into place. I then committed them to repo, only to discover the images were not displaying.

On the backend, GitHub pages handles resolving filenames in the repo to HTTPS paths. I am guessing that this system was on Linux, which is case-sensitive, and was still linking to the old file resource. I couldn't tell this just looking at the URL though because URLs, like macOS filenames, are case-insensitive...which is why is why it appeared to me that the correct file path was linked to but just wasn't updating.

In reality, there's some sort of intermediate cache that didn't refresh the filepath immediately. Eventually it caught up, and things started working as expected again.
